autogpt_platform/backend/backend/data/block.py

```diff
@@ -492,6 +510,135 @@ def get_blocks() -> dict[str, Type[Block]]:
     return load_all_blocks()
 
 
+def is_block_provider_configured(block_cls: type["Block"]) -> bool:
+    """
+    Check if a block has a valid authentication method configured at runtime.
+
+    For example if a block is an OAuth-only block and there env vars are not set,
+    do not show it in the UI.
+
+    """
+    from backend.sdk.registry import AutoRegistry
+
+    # Create an instance to access input_schema
+    try:
+        block = block_cls.create()
```

Comment:
`create()` is just a wrapper for `__init__`, but with a less well-defined type, causing type checking issues
```suggestion
        block = block_cls()
```
I have checked that `create()` isn't overridden by any block implementations